pyiron_base/job/template.py

```diff
@@ -35,6 +35,14 @@ def input(self):
     def output(self):
         return self._storage.output
 
+    def to_hdf(self, hdf=None, group_name=None):
+        HasStorage.to_hdf(self, hdf=self.project_hdf5)
+        GenericJob.to_hdf(self, hdf=hdf, group_name=group_name)
```

Comment:
```suggestion
        GenericJob.to_hdf(self, hdf=hdf, group_name=group_name)
        HasStorage.to_hdf(self, hdf=self.project_hdf5)
```
I suspect they should be changed around, because `GenericJob.to_hdf` should be messing with `self.project_hdft`.